mlflow/keras.py

```diff
@@ -150,33 +152,58 @@ def _is_tf_keras(model):
     elif type(keras_module) == str:
         keras_module = importlib.import_module(keras_module)
 
+    # check if path exists
```

Comment:
One small note on comments, we tend to prefer not adding them in favor of writing code that's easy to follow, reserving their use for cases where it's hard to tell exactly what's going on (e.g. here from the variable names & logic we can see that we're checking whether the path already exists). That's super fuzzy I know but hopefully it makes some sense - the idea is that comments can go out of date, so it's always better if the code is simple enough to follow on its own, but for more complex cases comments are definitely helpful

Reply:
Sounds good. I'll be sure to avoid superfluous comments in the future.